Doc/cmus.txt

```diff
@@ -1101,6 +1101,12 @@ mpris (true)
 output_plugin [roar, pulse, alsa, arts, oss, sndio, sun, coreaudio]
 	Name of output plugin.
 
+pl_env_vars
```

Comment:
I'm planning to write a more detailed section in the documentation for this.

Reply:
Also, we should consider setting a default value of `HOME` for the next major release.